scripts/watch_run.py

```diff
@@ -0,0 +1,34 @@
+# Copyright (c) Microsoft. All rights reserved.
+# Licensed under the MIT license. See LICENSE file in the project root for
+# full license information.
+import json
+import time
+import os
+from pygments import highlight
+from pygments.lexers import JsonLexer
+from pygments.formatters import TerminalFormatter
+import thief_secrets
+from thief_constants import Fields
+from azure.iot.hub import IoTHubRegistryManager
+
+registry_manager = IoTHubRegistryManager.from_connection_string(
+    thief_secrets.IOTHUB_CONNECTION_STRING
+)
+
+while True:
+    os.system("cls" if os.name == "nt" else "clear")
```

Comment:
Not sure what the equivalent command is on Windows, but for Linux I think `clear -x` would be preferable so running the script doesn't clear the scrollback.

Reply:
that's good to know.   If this was more than a niggly little script that probably won't get used, I would probably use the blessings library because it does a much better job with full screen CLI